docs/user-guide/settings/system-logs.md

```diff
@@ -0,0 +1,50 @@
+---
+description: 
```

Comment:
Should we add a description here or just drop it altogether?

Reply:
Damn good catch. It's needed and missing.